app/models/solidus_stripe/gateway.rb

```diff
@@ -87,19 +91,20 @@ def capture(amount_in_cents, payment_intent_id, options = {})
     # Authorizes and captures a certain amount on the provided payment source.
     # @todo add support for purchasing custom amounts
     def purchase(_amount_in_cents, source, options = {})
-      # Charge the Customer instead of the card:
-      payment_intent =
-        SolidusStripe::PaymentIntent.create_stripe_intent(
-          payment_method: source.payment_method,
-          order: options[:originator].order,
-          stripe_intent_options: options[:payment_intent_options] || {}
-        )
-
+      stripe_payment_intent = create_stripe_payment_intent(
+        source,
+        options[:originator].order,
+        stripe_payment_intent_options: {
+          capture_method: "manual",
+          confirm: true
+        }
+      )
+      captured_stripe_payment_intent = capture_stripe_payment_intent(stripe_payment_intent.id)
```

Comment:
It seems to me from the wording of the commit description that we're assuming that this method was not doing both authorization and capture before this change. 

Otherwise I'm not sure I understand why we're now doing the capture separately instead of at once with the default `capture_method`, which is _automatic_.

https://stripe.com/docs/api/payment_intents/create#create_payment_intent-capture_method